Dockerfile.ci

```diff
@@ -1,57 +1,74 @@
-FROM golang:1.12-stretch AS go-env
+# /bin/terraform
```

Comment:
Change `AS terraform` to `AS terraform-binary` so it's self-explanatory and remove comment.
Same for promtool below.  :)